flink-table/flink-table-common/src/test/java/org/apache/flink/table/catalog/CatalogTest.java

```diff
@@ -1130,8 +1151,9 @@ public void testListPartitionPartialSpec() throws Exception {
     @Test
     public void testGetTableStats_TableNotExistException() throws Exception {
         catalog.createDatabase(db1, createDb(), false);
-        exception.expect(org.apache.flink.table.catalog.exceptions.TableNotExistException.class);
-        catalog.getTableStatistics(path1);
+        assertThatThrownBy(() -> catalog.getTableStatistics(path1))
+                .isInstanceOf(
+                        org.apache.flink.table.catalog.exceptions.TableNotExistException.class);
```

Comment:
```suggestion
                .isInstanceOf(TableNotExistException.class);
```
This seems to be the same class as in other tests.